### ISSUE TITLE
Where can I download it?

### ISSUE THREAD

**Issue by create-xiaoyu** [Communi-Suggestu/Saecularia-Caudices#5]
The mod cannot be found on either CurseForge or Modrinth, and compiling it from the cloned source code was unsuccessful.

```Gradle
E:\Project\IDEA\Saecularia-Caudices>gradlew build
Calculating task graph as configuration cache cannot be reused because the file system entry '..\..\..\..\C:\Users\abc18\.gradle\caches\fabric-loom\1.21.1\layered\loom.mappings-layered+hash.845144970.jar' has been created.

> Configure project :fabric
Fabric Loom: 1.14.9
Failed to parse fabric.mod.json: E:\Project\IDEA\Saecularia-Caudices\fabric\src\main\resources\fabric.mod.json

> Task :core:neoFormMergeMappings FAILED
Exception in thread "main" java.lang.RuntimeException: Right does not exist: E:ProjectIDEASaecularia-Caudicescore.gradlecachesminecraftversions1.21.1client.txt
        at net.neoforged.installertools.Task.error(Task.java:32)
        at net.neoforged.installertools.ChainMappings.process(ChainMappings.java:92)
        at net.neoforged.installertools.ConsoleTool.main(ConsoleTool.java:55)

> Task :neoforge:neoFormMergeMappings FAILED
Exception in thread "main" java.lang.RuntimeException: Right does not exist: E:ProjectIDEASaecularia-Caudicesneoforge.gradlecachesminecraftversions1.21.1client.txt
        at net.neoforged.installertools.Task.error(Task.java:32)
        at net.neoforged.installertools.ChainMappings.process(ChainMappings.java:92)
        at net.neoforged.installertools.ConsoleTool.main(ConsoleTool.java:55)

[Incubating] Problems report is available at: file:///E:/Project/IDEA/Saecularia-Caudices/build/reports/problems/problems-report.html

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':core:neoFormMergeMappings'.
> Failed to execute stage: Default[outputs=[OutputEntry[output=E:\Project\IDEA\Saecularia-Caudices\core\build\neoForm\neoFormJoined1.21.1-20240808.144430\steps\mergeMappings\outputs.tsrg, isDirectory=false, createsDirectory=false, mergesDirectory=false]], execute=net.neoforged.gradle.common.runtime.tasks.DefaultExecute$$Lambda/0x000002648237d638@53724de]

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (powered by Develocity).
> Get more help at https://help.gradle.org.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':neoforge:neoFormMergeMappings'.
> Failed to execute stage: Default[outputs=[OutputEntry[output=E:\Project\IDEA\Saecularia-Caudices\neoforge\build\neoForm\neoFormJoined1.21.1-20240808.144430\steps\mergeMappings\outputs.tsrg, isDirectory=false, createsDirectory=false, mergesDirectory=false]], execute=net.neoforged.gradle.common.runtime.tasks.DefaultExecute$$Lambda/0x000002648237d638@3cf1f605]

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (powered by Develocity).
> Get more help at https://help.gradle.org.
==============================================================================

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.2.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
24 actionable tasks: 5 executed, 19 up-to-date
Configuration cache entry stored.
```
